dev/release/verify-release-candidate.sh

```diff
@@ -226,6 +226,7 @@ ${ARROW_CMAKE_OPTIONS:-}
 -DARROW_PYTHON=ON
 -DARROW_GANDIVA=ON
 -DARROW_PARQUET=ON
+-DPARQUET_REQUIRE_ENCRYPTION=ON
```

Comment:
This may make it harder for people to verify releases. We can see for the next major release